src/modules/popup.js

```diff
@@ -0,0 +1,110 @@
+          <div class="ProductPrice" id="product-price"> 
+              <p>${mealData[0].strTags}</p>
+              <p href="${mealData[0].strYoutube}">Youtube</p>
+              <p>${mealData[0].strIngredient1}</p>
+              <p>${mealData[0].strIngredient2}</p>
+          </div>
+      
+          <div class="ReviewTxtarea" id="review-txtarea">
+          <h3 id="review-count"></h3>    
+          <ul id="reviews-list"></ul>
+          </div>
+          <div>
+              <h3 id="give-review">Give Review</h3>
+              <form class="reviewForm" id="">
+                  <input id="form-name" type="text" placeholder="Your Name" required>
+                  <input id="form-review" type="text" placeholder="Give Review" required>
+                  <div class="style-popupbtn">
+                      <button type="button" class="button" id="submit-btn">
+                        Submit Review
+                      </button>
+                  </div>
+              </form>
+          </div>
+              <div id="counter"></div>
+            <p class="proj-primarytxt" id="proj-primarytxt-id"></p>
+            <div id="popuplist">
+              <ul class="no-bullets proj-list" id="tech-list-id">
+              </ul>
+              
+            </div>
+      </section>
+  </div>`;
+
+      overlay.innerHTML = div;
+      // Create DOM Elements for User Comments
+      const commentCount = document.getElementById('review-count');
+      let reviewCounts = commData.length;
+      if (!(reviewCounts > 0)) {
+        reviewCounts = 0;
+        commentCount.innerText = `Reviews (${reviewCounts})`;
+      } else {
+        commentCount.innerText = `Reviews  (${reviewCounts})`;
+        const Reviewul = document.getElementById('reviews-list');
+        commData.forEach((item) => {
+          const reviewDetail = document.createElement('li');
+          reviewDetail.innerText = `${item.creation_date} ${item.username}: ${item.comment}`;
+          reviewDetail.classList = 'review-detail';
+          Reviewul.append(reviewDetail);
+        });
+      }
+
+      // Close Button function for Popup
+      const closeButton = document.getElementById('x-icon-popup');
+      closeButton.addEventListener('click', () => {
+        if (overlay.style.display !== 'none') {
+          overlay.style.display = 'none';
+        }
+      });
+
+      setTimeout(() => {
+        const commSubmitBtn = document.getElementById('submit-btn');
+        const userName = document.getElementById('form-name');
+        const review = document.getElementById('form-review');
+        const itemId = IdMeal;
+        commSubmitBtn.addEventListener('click', (e) => {
+          e.preventDefault();
+          if (!(userName.value === '' || review.value === '')) {
+            PostReviewData(itemId, userName.value, review.value);
+            userName.value = '';
+            review.value = '';
+          }
+        });
+      }, 5000);
+    });
+};
```

Comment:
Kindly make sure that when you click on the submit button the popup window updates the list of comments and the counter automatically. This is part of the requirements of the project. :eyes: :eyes: 
![image](https://user-images.githubusercontent.com/98363075/197344524-f640d9d2-284c-49ed-930b-8921f5404fee.png)